Wox.Plugin.Runner/Wox.Plugin.Runner.csproj

```diff
@@ -18,7 +18,7 @@
     <OutputPath>..\Output\Debug\Wox.Plugin.Runner</OutputPath>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Flow.Launcher.Plugin" Version="1.1.0" />
+    <PackageReference Include="Flow.Launcher.Plugin" Version="1.4.0" />
     <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />

```

Comment:
I don't think the version is correct. Think we are on 2.1.0 currently

Reply:
emm I am not sure what you means. This is the plugin dependency version from flow, and the latest is 1.4.0.
 I will bump the version later.

Reply:
My bad, I misread it.